src/main/java/ogc/rs/apiserver/ApiServerVerticle.java

```diff
@@ -158,14 +168,7 @@ public void start() throws Exception {
       }
     });
 
-    HttpClientOptions httpCliOptions = new HttpClientOptions().setSsl(true);
```

Comment:
Is this related to the client running HTTP/HTTPS for S3 connection?

Reply:
When SSL is enabled by default, as far as I understand, non-SSL connections cannot be made by the `HttpClient` at all (I've not tested this).

In order to be flexible when making the request, to allow both HTTP and HTTPS, the `RequestOptions` class used in `DataFromS3` proved to be better suited, since according to the code at least, it enables SSL when `https` is part of the protocol - https://github.com/eclipse-vertx/vert.x/blob/27cb9611cd2e7eb34bed6486ca0131d661acb585/vertx-core/src/main/java/io/vertx/core/http/RequestOptions.java#L333